lib/rubocop/cop/style/empty_literal.rb

```diff
@@ -36,7 +38,8 @@ def on_send(node)
 
         def autocorrect(node)
           lambda do |corrector|
-            corrector.replace(replacement_range(node), correction(node))
+            result = correction(node)
+            corrector.replace(replacement_range(node), result) if result
```

Comment:
The check for `result` should be outside the `lambda`.
By `autocorrect` method returns a nil, RuboCop treats it as not being auto-corrected.

For example: https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/lint/safe_navigation_chain.rb#L48-L56